### ISSUE TITLE
fix: is_rockpi and is_raspberry_pi should not break outside Balena

### ISSUE THREAD

**Issue by marvinmarnold** [NebraLtd/hm-pyhelper#69]
**User Story**
As a user, I should not hit an exception if running outside Balena.

**Notes**
We [currently](is_raspberry_pi) detect hardware by inspecting the `BALENA_DEVICE_TYPE` environment variable, created by Balena. In test cases, this will raise an exception.

```
    def is_raspberry_pi():
        """
        Pulled from
        https://www.balena.io/docs/reference/base-images/devicetypes/
        """
        device_type = os.getenv('BALENA_DEVICE_TYPE')
        device_type_match = [
            'raspberry-pi2',
            'raspberrypi3',
            'raspberrypi3-64',
            'raspberrypi4-64',
            'nebra-hnt',
            'raspberrypicm4-ioboard'
        ]
    
        for device in device_type_match:
>           if device in device_type:
E           TypeError: argument of type 'NoneType' is not iterable
```

**Acceptance Criteria**
- is_rockpi and is_raspberry_pi should not raise exceptions if invoked outside Balena. Return False in both cases.

**Comment by shawaj** [comment]
@marvinmarnold for the version check, perhaps we could use `cat /proc/device-tree/model` when not on balena...

This gives the following output:

ROCK PI 4B
Raspberry Pi Compute Module 3 Plus Rev 1.0

**Comment by shawaj** [comment]
`/proc/cpuinfo` has this for RasPi too but not RockPi

**Comment by vpetersson** [comment]
> @marvinmarnold for the version check, perhaps we could use `cat /proc/device-tree/model` when not on balena...
> 
> This gives the following output:
> 
> ROCK PI 4B Raspberry Pi Compute Module 3 Plus Rev 1.0

Seems like a reasonable fallback.

**Comment by shawaj** [comment]
Partially solved by #84
